doc/source/whatsnew/v2.1.1.rst

```diff
@@ -42,7 +42,7 @@ Bug fixes
 
 Other
 ~~~~~
--
+- Enhancement in :meth:`_libs.tslibs.parsing.guess_datetime_format` to make public (:issue:`54727`)
```

Comment:
```suggestion
- :meth:`pandas.tseries.guess_datetime_format` is now available (:issue:`54727`)
```

Can you move this to the other enhancement section?

Reply:
Also this should be in 2.2.0.rst

Reply:
WIll do - thanks for the review!

Reply:
moved enhancement note to `2.2.0.rst` unde the **Other Enhancements** section ✅